geopandas/plotting.py

```diff
@@ -255,6 +255,11 @@ def plot_series(s, cmap=None, color=None, ax=None, figsize=None, **style_kwds):
                       "(for consistency with pandas)", FutureWarning)
         ax = style_kwds.pop('axes')
 
+    if not s.shape[0]:
+        warnings.warn("The GeoDataFrame you are attempting to plot is "
+                "empty. Nothing has been displayed.", UserWarning)
+        return ax
+
```

Comment:
I would put this just below the lines below (the check if `ax is None`), so an empty plot is returned.

Although that is of course debatable which of the two behaviours is preferred (empty ax or None). But personally an empty ax seems more consistent and less unexpected (eg if you then add some labels to the ax after the plotting call). And is also what matplotlib does.

Reply:
Good point! My thought was if you don't pass an `ax`, you should get nothing, but it makes sense that attempting to plot at least generates the axis object.

Reply:
For `plot_dataframe`, the creation of `ax` is below, would it make sense to move it up for this reason?

Reply:
yes